r2/r2/models/mail_queue.py

```diff
@@ -428,6 +437,7 @@ def utf8(s, reject_newlines=True):
             msg['To']      = utf8(self.to_addr)
             msg['From']    = utf8(fr)
             msg['Subject'] = utf8(self.subject)
+            msg['Date']    = utf8(self.get_date_string())
```

Comment:
It looks like the python standard library has a way to do this: https://docs.python.org/2/library/email.util.html#email.utils.formatdate

You could do something like:
```
timestamp = time.mktime(self.date.utctimetuple())
msg['Date'] = utf8(email.utils.formatdate(timestamp))
```

From some quick googling it's unclear whether we'd want the date to be in the server's timezone, or if just using UTC would be acceptable.